easybuild/easyconfigs/e/EasyBuild/EasyBuild-4.1.2.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'EB_EasyBuildMeta'
+
+name = 'EasyBuild'
+version = '4.1.2'
+
+homepage = 'https://easybuilders.github.io/easybuild'
+description = """EasyBuild is a software build and installation framework
+ written in Python that allows you to install software in a structured,
+ repeatable and robust way."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
should this be replaced with SYSTEM?

Reply:
@lexming No, not yet. This is done so people still using EasyBuild 3.x can easily upgrade to EasyBuild 4.x (`SYSTEM` is only known in EasyBuild v3.9.4, not in earlier EasyBuild 3.x releases).